core/src/main/java/org/mule/runtime/core/api/config/MuleConfiguration.java

```diff
@@ -41,8 +41,6 @@
 
     boolean isCacheMessageAsBytes();
```

Comment:
This is different functionality separate to keeping hold of the original message. This needs thinking about separately.